web-app/js/portal/details/StylePanel.js

```diff
@@ -8,15 +8,10 @@
 Ext.namespace('Portal.details');
 
 Portal.details.StylePanel = Ext.extend(Ext.Container, {
-
     constructor: function(cfg) {
         this.layer = cfg.layer;
 
-        var config = Ext.apply({
-            title: OpenLayers.i18n('stylePanelTitle')
-        }, cfg);
-
-        Portal.details.StylePanel.superclass.constructor.call(this, config);
+        Portal.details.LayerDetailsPanel.superclass.constructor.call(this, cfg);
```

Comment:
Yeah that's a mistake.  It worked (with this errors) because the superclass is Ext.Panel in both cases.